react-graphql-apollo-tutorial.md

```diff
@@ -2141,9 +2133,9 @@ const Organization = ({ organizationName }) => (
 export default Organization;
 {{< /highlight >}}
 
-As you can see, the Query component in the Organization component differs in three things: it takes a query tailored to the organization being the top level field of the query, it takes a variable to identify the organization, and it uses the newly introduced `skip` prop to skip executing the query in case no organization identifier is provided. Later on, you will pass an organization identifier from the App component. You may have noticed as well that the repository fragment, which you have introduced earlier for updating the local state in the cache, can be reused here. It saves you lines of code and you can be assured that the returned list of repositories will have the identical structure as the list of repositories in the Profile component.
+The Query component in the Organization component takes a query tailored to the organization being the top level field of the query.  It takes a variable to identify the organization, and it uses the newly introduced `skip` prop to skip executing the query if no organization identifier is provided. Later, you will pass an organization identifier from the App component.  It saves lines of code and ensures the returned list of repositories have identical structures to the list of repositories in the Profile component.
```

Comment:
"It saves lines of code and ensures the returned list of repositories have identical structures to the list of repositories in the Profile component." <- the essential part about the fragment is missing which makes this statement true. So it should be: "You may have noticed as well that the repository fragment, which you have introduced earlier for updating the local state in the cache, can be reused here. It saves lines of code and more importantly ensures that the returned list of repositories have identical structures to the list of repositories in the Profile component."

Reply:
Edited to:
You may have noticed that the repository fragment you introduced earlier to update the local state in the cache can be reused here. It saves lines of code, and more importantly, ensures the returned list of repositories have identical structures to the list of repositories in the Profile component.

Reply:
Sounds good!